src/nodes/cannonball.rs

```diff
@@ -0,0 +1,140 @@
+use macroquad::{
```

Comment:
Is cannonball going to be used for many weapons? I feel that it belong to a cannon item, not to the whole game?

Reply:
It could also appear individually from something like a “weapons volcano” and other spawn behaviors.